lib/src/strongly_connected_components.dart

```diff
@@ -52,12 +52,20 @@ List<List<T>> stronglyConnectedComponents<T>(
     lastVisited.addLast(node);
     onStack.add(node);
     // ignore: omit_local_variable_types
-    for (final T next in edges(node) ?? const []) {
+    for (final T next in edges(node)) {
       if (!indexes.containsKey(next)) {
         strongConnect(next);
-        lowLinks[node] = min(lowLinks[node], lowLinks[next]);
+        final n = lowLinks[node];
+        final x = lowLinks[next];
+        if (n != null && x != null) {
+          lowLinks[node] = min(n, x);
+        }
```

Comment:
This should just be:

```suggestion
        lowLinks[node] = min(lowLinks[node]!, lowLinks[next]!);
```

Reply:
done